modules/javafx.web/src/main/native/Source/WebKitLegacy/java/WebCoreSupport/HistoryItemClientJava.cpp

```diff
@@ -0,0 +1,22 @@
+#include "HistoryItemClientJava.h"
```

Comment:
Where did this file come from? I don't see any changes related to HistoryItemClient in the 2.44.4 release of WebKitGTK?

Presuming we need this file, it will need a proper copyright header.

Reply:
This file has been added in order to adapt to the changes to the interface added as part of webkit commit def02cb5c4c24ed8a05e4e5075eb6ec95216ef80 in HistoryItem.h.

Added the copyright header.

Reply:
Thanks.